docs/faq.md

```diff
@@ -0,0 +1,13 @@
+# Frequently Asked Questions
+
+## Why are you charging for the game on Steam and iOS when it is available for free?
+
+The game is being charged for on Steam and iOS because it is one way of supporting Anuke. You have the choice of getting the game for free or not, just like in Itch.io. You can choose to get the Steam version or get the regular desktop release. Unfortunately this is not possible on the App Store [anuke you will know this]
```

Comment:
This is a bit misleading. Unlike the free versions, the Steam version also has:
- A map browser
- A mod browser
- A schematic browser
- Seamless multiplayer (w/o port forwarding)
- Publically visible servers at startup
- Achievements
- Stat tracking
- Cloud saves / progress transfer
- A community/thread system

Reply:
Thank you for that! I haven't bought it on Steam yet (i have no money right now) but I might in the future, so thanks for clarifying those.